app/src/main/kotlin/com/simplemobiletools/flashlight/activities/WidgetBrightDisplayConfigureActivity.kt

```diff
@@ -51,13 +51,25 @@ class WidgetBrightDisplayConfigureActivity : ComponentActivity() {
                     onColorPressed = ::pickBackgroundColor,
                     onSavePressed = ::saveConfig
                 )
-            }
-        }
 
-        if (!isCustomizingColors && !isOrWasThankYouInstalled()) {
-            mFeatureLockedDialog = FeatureLockedDialog(this) {
-                if (!isOrWasThankYouInstalled()) {
-                    finish()
+
+                val featureLockedAlertDialogState = rememberAlertDialogState().apply {
+                    DialogMember {
+                        FeatureLockedAlertDialog(
+                            alertDialogState = this,
+                        ) {
+                            if (!isOrWasThankYouInstalled()) {
+                                finish()
+                            }
+                        }
+                    }
+                }
+                LaunchedEffect(isOrWasThankYouInstalled()) {
```

Comment:
`isOrWasThankYouInstalled` should come from `onStartEventValue`

check how `linkColor` was done